configure.ml

```diff
@@ -491,8 +495,27 @@ let check_caml_version () =
 
 let _ = check_caml_version ()
 
-let coq_debug_flag_opt =
-  if caml_version_nums >= [3;10] then coq_debug_flag else ""
+let findlib_version_list = numeric_prefix_list findlib_version
+
+let findlib_version_nums =
+  try
+    if List.length findlib_version_list < 2 then failwith "bad version";
+    List.map s2i findlib_version_list
+  with _ ->
+    die ("I found ocamlfind but cannot read its version number!\n" ^
+         "Is it installed properly?")
+
+let check_findlib_version () =
+  if findlib_version_nums >= [1;4;1] then
+    printf "You have OCamlfind %s. Good!\n" findlib_version
+  else
+    let () = printf "Your version of OCamlfind is %s.\n" findlib_version in
+    if !Prefs.force_findlib_version then
+      printf "*Warning* Your version of OCaml is outdated.\n"
```

Comment:
Did you mean `OCamlfind` instead of `OCaml` here?

Reply:
>Did you mean OCamlfind instead of OCaml here?

Fixed.